packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap

```diff
@@ -2036,7 +2036,7 @@ export default function NestedJson(props) {
         }}
         currentFieldValue={currentNicknamesValue}
         label={\\"nick-Names2\\"}
-        items={nicknames}
```

Comment:
1. shouldn't the variable name still be `nicknames`?
2. what do you think about doing this in the reset function instead? Something like `setNicknames(cleanedValue.nickNames ?? [])`. If too big of a change, maybe not worth it.

Reply:
1. Yes it should
2. Yeah thats is possible